config/formats.js

```diff
@@ -1512,7 +1516,8 @@ exports.Formats = [
 	},
 	{
 		name: "[Gen 1] OU (tradeback)",
-
+      desc: [ Generation 1 OU with movepool additions from the Time Capsule.
```

Comment:
Extra space before `desc`, first sentence needs to be surrounded on quotes and should be on its own line separate from `desc: [`, and dont forget the comma after the closing quote for the sentence. I also don't see why you can't just say "RBY OU"